Add international telephone field

derekcaneja opened the pull request:
Add international telephone field for easily dealing with entering international numbers in meteor.
Use ipinfo.io to pre-populate the dial code input with the dial code of the user's country.

derekcaneja committed:
Refactor package and add component.

Integrate ipinfo.io for pre-populating the dial code input with the cial code of the user's country.
Add input component for easily dealing with entering mobile numbers in meteor.
Update README.md to reflect code changes.

mikedemarais commented on components/phone_input.css:
```diff
@@ -0,0 +1,26 @@
+input {
```
Should be `.dial-code, .phone-number { ... }`

jperl commented on PhoneFormat.js:
```diff
@@ -1,524 +1,441 @@
-countryForE164Number("US", "+18646978257");
- Returns: US
+// Get the user's current country based on the user's ip address
+Phone._getCountryFromIP = function (callback) {
```
`ForIp`

jperl commented on PhoneFormat.js:
```diff
@@ -1,524 +1,441 @@
- Returns: US
+// Get the user's current country based on the user's ip address
+Phone._getCountryFromIP = function (callback) {
+  var country = localStorage.getItem('current_country');
```
Prefix this to not run into other people's local storage. Like `pf.country_code`

derekcaneja committed:
Prefix cookie, change function name, and update css.

derekcaneja committed:
Add ability to get a country name based on a dial code.

Ex.
'+1' => 'United States'
'+852' => 'Hong Kong'

Allows us to update the number format if the user changes the dial code manually.

jperl commented on phone.js:
```diff
@@ -0,0 +1,686 @@
+  return phone;
+};
+
+var countryCodeToNameArr = new Array();
```
Change to object.

derekcaneja committed:
Switch arrays to objects.

jperl commented:
:+1:
